zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/config/ZakenApiAutoConfiguration.kt

```diff
@@ -70,4 +71,12 @@ class ZakenApiAutoConfiguration {
             zaakInstanceLinkRepository,
         )
     }
+
+    //TODO: make sure this does not cause a conflict with the existing bean in the openzaak module
```

Comment:
fix TODO

Reply:
Fixed by deleting the `ZaakInstanceLinkService` in `openzaak` module

Reply:
This is a breaking change. I've created a documentation PR